frontend/src/styles.css

```diff
@@ -6,4 +6,8 @@ html, body{
   height: 100%;
 }
 
+.appContainer{
```

Comment:
That makes sense - that's also how it's done in the `react-router` readme example